test/dummy/app/javascript/application.js

```diff
@@ -1,2 +1,4 @@
 // Configure your import map in config/importmap.rb. Read more: https://github.com/rails/importmap-rails
-import "@hotwired/turbo-rails"
+import { Turbo } from "@hotwired/turbo-rails"
+
+window.StreamActions = Turbo.StreamActions
```

Comment:
Should this just be `window.Turbo = Turbo`?

Reply:
This change fails `test/system/broadcasts_test.rb`:

```diff
--- a/test/dummy/app/javascript/application.js
+++ b/test/dummy/app/javascript/application.js
@@ -1,4 +1,4 @@
 // Configure your import map in config/importmap.rb. Read more: https://github.com/rails/importmap-rails
-import { Turbo } from "@hotwired/turbo-rails"
+import { Turbo } "@hotwired/turbo-rails"
 
-window.StreamActions = Turbo.StreamActions
+window.Turbo = Turbo
```